shared-helpers/src/programHelpers.ts

```diff
@@ -0,0 +1,58 @@
+import { ApplicationProgram, Program } from "@bloom-housing/backend-core/types"
+
+export const PROGRAMS_FORM_PATH = "application.programs"
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const mapProgramsToApi = (programs: Program[], data: Record<string, any>) => {
+  if (Object.keys(data).length === 0) return []
+
+  const savedPrograms = [] as ApplicationProgram[]
+  Object.entries(data).forEach(([key, value]) => {
+    const program = programs.find((item) => item.formMetadata?.key === key)
+    let claimed = true
+    const options = []
+
+    if (value == "none") {
+      claimed = false
+      program?.formMetadata?.options.forEach((option) => {
+        options.push({
+          key: option.key,
+          checked: false,
+          extraData: [],
+        })
+      })
+    } else {
```

Comment:
under what circumstances does value === "none"?

Reply:
Whoops, I think this was left here by mistake after my latest refactor. I'll clean it up.